spec/features/page_feature_spec.rb

```diff
@@ -135,7 +135,11 @@
       it "a form and button to logout of alchemy" do
         within('#alchemy_menubar') do
           expect(page).to \
-            have_selector("li form[action='#{Alchemy.logout_path}'], li button[type='submit']")
```

Comment:
I think this selector was wrong 🤔 . Since it is a css selector in the format of `A, B`, it will match against any element A **or** B. So `have_seletor` passes if at least one of `li form[action='#{Alchemy.logout_path}']` **or** `li button[type='submit']` is present. I think that the test should cover `li form[action='#{Alchemy.logout_path}']` **and** `li button[type='submit']")`. This is why I splitted the selector in mutiple `have_selector` expectations.

I hope this helps :)